NOTICE.txt

```diff
@@ -4,3 +4,6 @@ Copyright 2007-2024 The Apache Software Foundation
 This product includes software developed at
 The Apache Software Foundation (https://www.apache.org/).
 
+This product includes images from Unsplash provided under
+the Unsplash license (https://unsplash.com/license).
+These files are used as test data.
```

Comment:
Breaks the build.

Run 'mvn' alone before you push.

Files should end with an empty line.

Reply:
Fixed.